model/export/AllBookletsExport.php

```diff
@@ -0,0 +1,843 @@
+<?php
+/**
+ * Copyright (c) 2017 (original work) Open Assessment Technologies SA.
+ */
+
+namespace oat\taoResultExports\model\export;
+
+use oat\oatbox\filesystem\File;
+use oat\oatbox\filesystem\FileSystemService;
+use oat\oatbox\service\ServiceManager;
+use oat\taoDelivery\model\execution\ServiceProxy;
+use oat\taoDeliveryRdf\model\DeliveryAssemblyService;
+use oat\taoResultServer\models\classes\ResultManagement;
+use oat\taoResultServer\models\classes\ResultServerService;
+use qtism\data\AssessmentTest;
+use qtism\data\ExtendedAssessmentItemRef;
+use qtism\data\state\OutcomeDeclaration;
+use qtism\data\state\ResponseDeclaration;
+use qtism\data\storage\php\PhpDocument;
+
+class AllBookletsExport
```

Comment:
Could u implements ServiceLocatorAwareInterface or extends Configurable to avoid ServiceManager::getServiceManager